projects/packages/forms/src/contact-form/class-contact-form-endpoint.php

```diff
@@ -9,12 +9,406 @@
+		);
+
+		$schema['properties']['fields'] = array(
+			'description' => __( 'The custom form fields and their submitted values.', 'jetpack-forms' ),
+			'type'        => 'string',
+			'context'     => array( 'view', 'edit', 'embed' ),
+			'arg_options' => array(
+				'sanitize_callback' => 'sanitize_text_field',
+			),
+			'readonly'    => true,
+		);
+
+		$this->schema = $schema;
+
+		return $this->add_additional_fields_schema( $this->schema );
+	}
+
+	/**
+	 * Prepares the item for the REST response.
+	 *
+	 * @param object          $item    WP Cron event.
+	 * @param WP_REST_Request $request Request object.
+	 * @return WP_REST_Response Response object on success.
+	 */
+	public function prepare_item_for_response( $item, $request ) {
+		$response = parent::prepare_item_for_response( $item, $request );
+		$data     = $response->get_data();
+		$fields   = $this->get_fields_for_response( $request );
+
+		$base_fields   = array(
+			'email_marketing_consent' => '',
+			'entry_title'             => '',
+			'entry_permalink'         => '',
+			'feedback_id'             => '',
+		);
+		$data_defaults = array(
+			'_feedback_author'       => '',
+			'_feedback_author_email' => '',
+			'_feedback_author_url'   => '',
+			'_feedback_all_fields'   => array(),
+			'_feedback_ip'           => '',
+			'_feedback_subject'      => '',
+		);
+
+		$feedback_data = array_merge(
+			$data_defaults,
+			\Automattic\Jetpack\Forms\ContactForm\Contact_Form_Plugin::parse_fields_from_content( $item->ID )
+		);
+
+		$all_fields = array_merge( $base_fields, $feedback_data['_feedback_all_fields'] );
+
+		$data['date'] = get_the_date( 'c', $data['id'] );
+		if ( rest_is_field_included( 'uid', $fields ) ) {
+			$data['uid'] = $all_fields['feedback_id'];
+		}
+		if ( rest_is_field_included( 'author_name', $fields ) ) {
+			$data['author_name'] = $feedback_data['_feedback_author'];
+		}
+		if ( rest_is_field_included( 'author_email', $fields ) ) {
+			$data['author_email'] = $feedback_data['_feedback_author_email'];
+		}
+		if ( rest_is_field_included( 'author_url', $fields ) ) {
+			$data['author_url'] = $feedback_data['_feedback_author_url'];
+		}
+		if ( rest_is_field_included( 'author_avatar', $fields ) ) {
+			$data['author_avatar'] = empty( $feedback_data['_feedback_author_email'] ) ? '' : get_avatar_url( $feedback_data['_feedback_author_email'] );
+		}
+		if ( rest_is_field_included( 'email_marketing_consent', $fields ) ) {
+			$data['email_marketing_consent'] = $all_fields['email_marketing_consent'];
+		}
+		if ( rest_is_field_included( 'ip', $fields ) ) {
+			$data['ip'] = $feedback_data['_feedback_ip'];
+		}
+		if ( rest_is_field_included( 'entry_title', $fields ) ) {
+			$data['entry_title'] = $all_fields['entry_title'];
+		}
+		if ( rest_is_field_included( 'entry_permalink', $fields ) ) {
+			$data['entry_permalink'] = $all_fields['entry_permalink'];
+		}
+		if ( rest_is_field_included( 'subject', $fields ) ) {
+			$data['subject'] = $data['_feedback_subject'];
```

Comment:
I am seeing this warning in the logs:

```[26-Feb-2025 12:20:57 UTC] PHP Warning:  Undefined array key "_feedback_subject" in /usr/local/src/jetpack-monorepo/projects/packages/forms/src/contact-form/class-contact-form-endpoint.php on line 284
```

Reply:
Thanks for pushing the fix!

Reply:
https://github.com/Automattic/jetpack/pull/41602/commits/86f4c439620368d97584b1cc4a10849e7378289d